Source/KSCrash/Source/KSCrash/Recording/BSG_KSCrash.m

```diff
@@ -226,6 +228,9 @@ - (BOOL)install {
         return false;
     }
 
+    // Maintain a cache of info about dynamically loaded binary images
+    [self listenForLoadedBinaries];
```

Comment:
Potentially worth putting this before `bsg_kscrash_install`, or at least initialising an empty array of binary images to prevent against the case where a crash occurs during/at the same time as initialisation.